docs/docs/cmd/spo/list/list-sitescript-get.md

```diff
@@ -35,3 +38,18 @@ located in site _https://contoso.sharepoint.com/sites/project-x_
 ```sh
 m365 spo list sitescript get --webUrl https://contoso.sharepoint.com/sites/project-x --listId cc27a922-8224-4296-90a5-ebbc54da2e85
 ```
+
+Extract a site script from an existing SharePoint list with server relative url _sites/project-x/Documents_
```

Comment:
and the same here to remove the values from the example line and make example description more general